types/comtac/Cluey/uplink.js

```diff
@@ -1713,6 +2519,19 @@ function decodePingPayload(data) {
   };
 }
 
+function checkTimestamp(timestamp) {
+  let now = new Date();
+  now = now.getTime();
+  const sampleTime = timestamp.getTime();
+
+  if (sampleTime < now - 2629746000 || sampleTime > now) {
```

Comment:
Nitpick, you could use a variable so to make it clear what this value is. E.g.

`let secondsPerMonth = 2629746000;`

And possibly make it clear what a month is.